api/models.py

```diff
@@ -1,3 +1,32 @@
 from django.db import models
+from django.utils import timezone
+from django.contrib.auth.models import User
 
-# Create your models here.
+ACTIVITY_LEVEL = (
+			("SC", "Sleepy_Cat"),
+			("LC", "Lazy_Cat"),
+			("AC", "Active_Cat"),
+			("MC", "Mentor_Cat"),
+	)
+
+class UserProfile(models.Model):
+	user = models.OneToOneField(User) 
+	#id = models.AutoField(primary_key=True) 
+	fullname = models.CharField(max_length=100, blank=False)
+	githubuser = models.CharField(max_length=100, null=False, blank=False)
+	email = models.EmailField(blank=False)
+	profile_picture = models.ImageField(blank = True)  
+	website = models.URLField(null=False)
+	bio = models.TextField(max_length=2000,blank=True)
+	active = models.BooleanField(default=False)
+	activity_status = models.CharField(max_length=2, choices=ACTIVITY_LEVEL)
+	last_logged_in = models.DateTimeField(auto_now=True, blank=False)
+	date_created = models.DateTimeField(auto_now_add=True, blank=False)
+	date_updated = models.DateTimeField(auto_now_add=True)
+
+	def publish(self):
```

Comment:
When i was reading the documentation link Alja sent all made sense to me, therefore the method was written to my code. Will deleted bot of them, thx